lib/styles/components/_buttons.scss

```diff
@@ -3,48 +3,183 @@
 @import '../settings/spacings';
 @import '../settings/radiuses';
 @import '../settings/typography';
+@import '../settings/buttons';
 
-$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black, 0.35);
+$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black-rgb, 0.35);
 
-@mixin outlinedColor($color) {
-	color: rgba($color, 1);
+@mixin setButtonColor($color: '', $color-hovered: '', $color-focused: '', $color-disabled: '') {
+	@if $color != '' {
```

Comment:
you could get rid of `ifs` if you use `null` instead of empty string: https://blog.teamtreehouse.com/smarter-sass-mixins-null

Reply:
It's good news. I replaced all '' to null in `Button`.

Reply:
you changed default value to `null` but `ifs` are still in place. Do you want to keep them in the code for readibility?

Reply:
Having `if` in the code we generating less final code. I know is very small improvement. I have no problem with these `if`.

Reply:
@kvas-damian I removed redundant `if` conditions. By the way, I changed the `BadgeScore`. I know it doesn't fit into this PR, but I didn't want to create a separate one for such a small thing.